frontend-react/src/hooks/filters/UseSortOrder.ts

```diff
@@ -3,6 +3,8 @@ import { Dispatch, useReducer } from "react";
 enum SortSettingsActionType {
     CHANGE_COL = "change-column",
     SWAP_ORDER = "swap-order",
```

Comment:
@kevinhaube what exactly do we mean by "local" here, generally?

Reply:
The state to manage local sorting (sorting of the API response array on the front-end) is decoupled from the state that manages API sort, so the actions had to be decoupled as well. Anything updating the front-end sorting is labeled local